### ISSUE TITLE
Capacitor v4.7 Importing WebPlugin

### ISSUE THREAD

**Issue by Sebastian-Elstadt** [unitree-czk/capacitor-udp#13]
My project is running Ionic v6.1.9 with Angular v15.2.0 and Capacitor v4.7.0.

In my service I am trying to import the UdpPlugin like:
```import { UdpPlugin } from 'capacitor-udp';```

But this UdpPlugin class is of type `UdpPluginWeb`. Which does not have the create methods. If I go to definition, it takes me to the web.d.ts file in the capacitor-udp/dist/esm/ folder in node_modules and I see the `UdpPlugin` class gets export as:
```
declare const UdpPlugin: UdpPluginWeb;
export { UdpPlugin };
```

Plugins and PluginRegistery from @capacitor/core is now deprecated so I can't follow the fixes specified in this plugin's readme file. For Capacitor v4 you must import the plugins directly. What can I do to fix this?

**Comment by gee1k** [comment]
Hi, I'm having the same problem, do you have a solution?

**Comment by Sebastian-Elstadt** [comment]
No, I abandoned the plugin and opted for a TCP approach using WebSockets.

**Comment by Sebastian-Elstadt** [comment]
> Hi, I'm having the same problem, do you have a solution?

No, I abandoned the plugin and opted for a TCP approach using WebSockets.
